velox/functions/sparksql/String.h

```diff
@@ -1130,4 +1130,59 @@ struct ReplaceFunction {
   }
 };
 
+template <typename T>
+struct FindInSetFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool callNullFree(
+      out_type<int32_t>& result,
+      const null_free_arg_type<Varchar>& str,
+      const null_free_arg_type<Varchar>& strArray) {
+    return doCall(result, str, strArray);
+  }
+
+  FOLLY_ALWAYS_INLINE bool
+  doCall(out_type<int32_t>& result, StringView match, StringView strArray) {
+    int32_t index = 1;
+    int32_t lastComma = -1;
+
+    auto _match = match.str();
```

Comment:
It doesn't satisfy the naming convention.
See
https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions